### DISCUSSION TITLE
解决使用 profile-sync-daemon 时 Firefox 开机启动报错

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#353] [Comments]
# 解决使用 profile-sync-daemon 时 Firefox 开机启动报错

解决使用 profile-sync-daemon 时开机报错 "Firefox is already running, but is not responding"

https://ouuan.moe/post/2024/05/psd-firefox-startup

**Comment by Nikaidou-Shinku** [comment]
~~内存用不完后我的选择是去使用 zfs，然后我喜提 OOM 了~~
